src/services/invoice.services.ts

```
@@ -152,6 +159,57 @@ export const getInvoiceWithMerchant = async (merchantId: string, id: string) =>
+  }
+
+  if (data.amount !== undefined) {
+    const amount = parseAmount(data.amount);
+    if (amount === null) {
+      throw new AppError(400, 'amount must be a positive integer');
+    }
+    updateData.amount = amount;
+  }
+
+  if (data.description !== undefined) {
+    const description = typeof data.description === 'string' ? data.description.trim() : data.description;
```

ESLint

[error] Insert `⏎·····`

(prettier/prettier)